### PR TITLE
Clean up BuildFiles under Utilities/

### PR DESCRIPTION
**Pull request by fwyzard**
Clean up BuildFile.xml filea with `xmllint --format`.

### CODE REVIEW

**Comment by fwyzard** [comment]
@cmsbuild, please test

**Comment by Dr15Jones** [comment]
@smuzaffar any idea what the error message is from?
```ERROR: empty variable. PR had errors - #29736.```

File: Utilities/StorageFactory/test/BuildFile.xml [hunk 1]
```diff
@@ -1,39 +1,18 @@
...
-</bin>
-<bin   file="rfio2.cpp" name="test_StorageFactory_Rfio2">
-</bin>
-<bin   file="rfio3.cpp" name="test_StorageFactory_Rfio3">
-</bin>
-<bin   file="write.cpp" name="test_StorageFactory_Write">
-  <flags NO_TESTRUN="1"/>
-</bin>
-<bin   file="mkstemp.cpp" name="test_StorageFactory_Mkstemp">
+</bin><bin file="local2.cpp" name="test_StorageFactory_Local2">
```
Comment (smuzaffar) [inline Utilities/StorageFactory/test/BuildFile.xml:1]: This does not look correct. @Dr15Jones

Comment (Dr15Jones) [reply Utilities/StorageFactory/test/BuildFile.xml:1]: Good catch! I'm guessing this is caused by the fact that `BuildFile.xml` files in general are not actually fully XML conformant (i.e. XML requires one and only one top level node while the `BuildFile.xml` format does not.)

Comment (fwyzard) [reply Utilities/StorageFactory/test/BuildFile.xml:1]: The comment says

> we want to wait until the framework decides on a threading model to implement a fix

maybe it's obsolete ?

**Comment by fwyzard** [comment]
@cmsbuild, please test

**Comment by civanch** [comment]
+1

**Comment by silviodonato** [comment]
merge

File: Utilities/StorageFactory/test/BuildFile.xml (hunk 1 above)
Comment (makortel) [reply Utilities/StorageFactory/test/BuildFile.xml]: > The comment says
> 
> > we want to wait until the framework decides on a threading model to implement a fix
> 
> maybe it's obsolete ?

The test was disabled in https://github.com/cms-sw/cmssw/commit/e09f0ac72480f91cc995fb8e1fcdc39bfbd79ac7. @Dr15Jones Would you be able to tell if there is anything that would prevent the test to run?

**Comment by makortel** [comment]
+1

post-merge

**Comment by santocch** [comment]
+1

**Pull request merged**